src/hooks/useHtmlPaste/index.ts

```diff
@@ -46,9 +45,20 @@ const useHtmlPaste: UseHtmlPaste = (textInputRef, preHtmlPasteCallback, removeLi
                 insertByCommand(text);
             }
 
+            if (!textInputRef.current?.isFocused()) {
+                textInputRef.current?.focus();
+                return;
+            }
+
             // Pointer will go out of sight when a large paragraph is pasted on the web. Refocusing the input keeps the cursor in view.
-            textInputRef.current?.blur();
-            textInputRef.current?.focus();
+            // to avoid the keyboard in mobile web if using blur() and focus() function, we just need to dispatch the event to trigger the onFocus handler
```

Comment:
```suggestion
            // To avoid the keyboard in mWeb if using blur() and focus() functions, we need to dispatch the event to trigger the onFocus handler
```